ci/me_regress.sh

```diff
@@ -12,39 +12,33 @@ elif [ $1 == "verilator" ]
 then
     SUFFIX=sc
 else
-  echo "Usage: $0 <verilator, vcs> <testlist> [num_cores]"
+  echo "Usage: $0 <verilator, vcs> <testlist>"
   exit 1
 fi
 
-# Default to 1 core
-N=${2:-1}
-
-# Bash array to iterate over for configurations
-cfgs=(
-    "e_bp_half_core_cfg"
-    "e_bp_half_core_ucode_cce_cfg"
+# Bash array to iterate over for coherence protocols for ucode CCE tests
+protos=(
+    "ei"
+    "msi"
+    "mesi"
+    "msi-nonspec"
+    "mesi-nonspec"
     )
 
 # The base command to append the configuration to
 cmd_base="make -C bp_me/syn run_testlist.${SUFFIX}"
 
 # Any setup needed for the job
 
-let JOBS=${#cfgs[@]}
-let CORES_PER_JOB=${N}/${JOBS}+1
+let JOBS=${#protos[@]}
```

Comment:
Maybe do the min of N and #protos, here?  It might surprise users to think they're running with 2 threads and have 5+ threads spawn

Reply:
nvm